test/modules/ecs6-class/line.test.js

```diff
@@ -0,0 +1,79 @@
+ const Line =require('../../../modules/ecs6-class/line')
+ const Point = require('../../../modules/ecs6-class/point')
+const{calculateSlope}=require('../../../modules/ecs6-class/line')
+
+const mockConstructor1=jest.fn(constructor)
+const mockConstructor=jest.fn(constructor)
+const mockCalculateSlope=jest.fn(calculateSlope)
```

Comment:
this way of mocks is total wrong 
mock classes the right way and restore the mocks at the end of the tests